torchtune/utils/ppo_utils/kl_controller.py

```diff
@@ -0,0 +1,56 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+# All rights reserved.
+#
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+
+import numpy as np
+
+
+class AdaptiveKLController:
```

Comment:
@ebsmothers  Should this be an nn.Module?

Reply:
This has been dropped in more recent implementations (e.g. https://arxiv.org/abs/2204.05862), so I omitted it.